c10/core/impl/alloc_cpu.h

```diff
@@ -4,6 +4,11 @@
 
 #include <cstddef>
 
+#ifdef __linux__
+#include <sys/mman.h>
+#include <unistd.h>
+#endif
```

Comment:
Can you please move this one to alloc_cpu.cpp (i.e. include what you use, this header does not need those includes)

Reply:
Hi @malfet , I have rebased the PR and also addressed this comment.